src/OptPropManager.cc

```diff
@@ -214,13 +220,23 @@ void OptPropManager::ReadValuesFromFile(const G4String& filename, std::vector<G4
 	
 	
 	while(getline(infile,str)){
+		
+    if(fVerbose>=OptPropManager::kDebug){
+      std::cout << "\tDebug --> OptPropManager::ReadValuesFromFile(...): processing line <" << str << ">." << std::endl;
+    }
+		
 		ss_tmp.clear(); ss_tmp.str("");
 		ss_tmp << str;
 		
 		ss_tmp >> str;
+    
+    if(fVerbose>=OptPropManager::kDebug){
+      std::cout << "\tDebug --> OptPropManager::ReadValuesFromFile(...): processing line <" << str << ">." << std::endl;
+    }
+		
 		G4double ph_en_d = std::stod(str);
 		
-		if(ss_tmp){//There is only one value while the file format is defined with 2 columns
```

Comment:
I'm not entirely sure about the functionality of that line. However, I need to negate the if condition in order to make it read the values from the file.

Reply:
Yes, I believe you are right. This was one of the first a fix that I made. I guess that something went wrong in very first commits of the code. Anyway, you should negate that.